web_external/templates/StudyResults/studyResultsPage.jade

```diff
@@ -40,13 +40,9 @@
           #isic-study-results-global-features-container
 
         .tab-pane#isic-study-results-local-features-tab-content
-          #isic-study-results-local-features-container
 
       // Container for images associated with tabs. Visibility is controlled by logic in StudyResultsView.js.
       .isic-study-results-flex-grow.isic-study-results-image-container
 
-        // Local features image container
```

Comment:
It makes sense that this should not be outside the local features view. But the reason for this is that I couldn't get the layout to work otherwise in conjunction with the Bootstrap tabs. I realize the styling isn't complete on this WIP, but I can't be convinced that this restructuring is okay until the layout and interaction work as before.